opencensus-shim/src/main/java/io/opentelemetry/opencensusshim/OpenTelemetryNoRecordEventsSpanImpl.java

```diff
@@ -0,0 +1,68 @@
+package io.opentelemetry.opencensusshim;
```

Comment:
This file is missing a licence header, and I'm actually not sure what should we put here, since parts of the code were copied from OpenCensus IIUC?

Reply:
We have this pattern for including two licenses, yeah in any file with copies let's use it

https://github.com/open-telemetry/opentelemetry-java/blob/master/context/src/main/java/io/opentelemetry/context/Context.java#L6